core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractProject.java

```diff
@@ -90,29 +93,34 @@
   @GeneratedValue(strategy = GenerationType.AUTO)
   private long projectId = AbstractProject.UNSAVED_ID;
 
-  @Transient
+  @OneToMany(targetEntity = SampleImpl.class, fetch = FetchType.LAZY, mappedBy = "project")
   private Collection<Sample> samples = new HashSet<>();
-  @Transient
-  private Collection<Run> runs = new HashSet<>();
-  @Transient
+  @ManyToMany(targetEntity = StudyImpl.class, fetch = FetchType.LAZY)
+  @JoinTable(name = "Project_Study", joinColumns = { @JoinColumn(name = "studies_studyId") }, inverseJoinColumns = {
+      @JoinColumn(name = "Project_projectId") })
   private Collection<Study> studies = new HashSet<>();
-  @Transient
+  @OneToMany(targetEntity = ProjectOverview.class, mappedBy = "project")
   private Collection<ProjectOverview> overviews = new HashSet<>();
-  @Transient
+  @ElementCollection
+  @JoinTable(name = "Project_Issues", joinColumns = { @JoinColumn(name = "project_projectId") })
+  @Column(name = "issueKey")
   private Collection<String> issueKeys = new HashSet<>();
 
-  @Column
-  @Type(type = "progressTypeUserType")
+  @Enumerated(EnumType.STRING)
   private ProgressType progress;
 
-  @OneToOne(targetEntity = ReferenceGenomeImpl.class)
+  @ManyToOne(targetEntity = ReferenceGenomeImpl.class)
   @JoinColumn(name = "referenceGenomeId", referencedColumnName = "referenceGenomeId", nullable = false)
   private ReferenceGenome referenceGenome;
 
-  @Transient
+  @ManyToOne
```

Comment:
It's not many with respect to project, but the things in the project may use the same security profile. Globally, security profiles are going to have to be `ManyToOne`, I think.

The `targetEntity` isn't necessary because the field type is a class and not a collection.